packages/components/breadcrumb/src/breadcrumb.vue

```diff
@@ -9,34 +9,28 @@
   </div>
 </template>
 
-<script lang="ts">
-import { defineComponent, provide, ref, onMounted } from 'vue'
+<script lang="ts" setup>
+import { provide, ref, onMounted } from 'vue'
 import { elBreadcrumbKey } from '@element-plus/tokens'
```

Comment:
```suggestion
import { breadcrumbKey } from '@element-plus/tokens'
```
Please rename token name